library/public/camera.h

```diff
@@ -57,6 +57,7 @@ class F3D_EXPORT camera
    * Angles are in degrees.
    */
   virtual camera& dolly(double val) = 0;
+  virtual camera& pan(double right, double up) = 0;
```

Comment:
I wonder if dolly and pan should be separately documented to clarify the arguments

Reply:
would changing `dolly(double val)` to `dolly(double forward)` be self-explanatory enough?

Reply:
definitely an improvement.

Reply:
unrelated to that PR though